lib/torba/verify.rb

```diff
@@ -1,7 +1,14 @@
 require "torba"
 
-if STDOUT.tty?
-  Torba.pretty_errors { Torba.verify }
-else
-  Torba.verify
+# We purposely skip verification during Rake execution, because Rake is used to
+# run the Torba:RakeTask during the "assets:precompile" stage of deployment. In
+# this case it is OK that the Torbafile hasn't been packed yet, because we are
+# about to do so.
+#
+unless defined?(Rake::Task)
```

Comment:
Given that `bin/rails` can be used, then we can't rely on `$0`. And if `bin/rake` or `bin/rails` binstubs are used, then Rails is loaded *before* Rake is loaded, so we can't rely on `defined?(Rake)`.

The only thing I can think of is checking `ARGV.include?("assets:precompile")`. That would mean the logic becomes:

```ruby
unless defined?(Rake::Task) || ARGV.include?("assets:precompile")
  # verify ...
end
```

Should I go with that?